### PR TITLE
Remove deprecated code.

### PR DESCRIPTION
**Pull request by OleksiiZubko**
Removing deprecated code which we've moved
to the core component 6months ago.

Adapted tests.

Resolves: OLPEDGE-2466

Signed-off-by: Oleksii Zubko <ext-oleksii.zubko@here.com>

### CODE REVIEW

File: @here/olp-sdk-core/lib/utils/TileKey.ts [hunk 1]
```diff
@@ -238,6 +238,30 @@ export class TileKey implements QuadKey {
...
+        // tslint:disable-next-line:no-magic-numbers
+        if (key.level < 0 || key.level > 26) {
+            return false;
+        }
+        const dimensionAtLevel = Math.pow(2, key.level);
+        const rowValid = key.row >= 0 && key.row < dimensionAtLevel;
+        const columnValid = key.column >= 0 && key.column < dimensionAtLevel;
+        return rowValid && columnValid;
+    }
+
```
Comment (andescu) [inline @here/olp-sdk-core/lib/utils/TileKey.ts:264]: Either add this to the commit message or split into a separate commit as this does not look like removal of deprecated code.

Comment (OleksiiZubko) [reply @here/olp-sdk-core/lib/utils/TileKey.ts]: will create separate MR

**Review by andescu — APPROVED** [review]

**Review by mykhailo-kuchma — APPROVED** [review]
